Files/Views/ModernShellPage.xaml.cs

```diff
@@ -197,6 +200,18 @@ public ModernShellPage()
             AppServiceConnectionHelper.ConnectionChanged += AppServiceConnectionHelper_ConnectionChanged;
         }
 
+        private void FolderSettings_LayoutPreferencesUpdateRequired(object sender, LayoutPreferenceEventArgs e)
+        {
+            if (FilesystemViewModel != null)
+            {
+                FolderSettingsViewModel.UpdateLayoutPreferencesForPath(FilesystemViewModel.WorkingDirectory, e.LayoutPreference);
```

Comment:
@duke7553 could you take `FolderSettingsViewModel` from `e` and do the same on ln210?

Reply:
@d2dyno1 The latest commit uses sender for calling the view model method.

Reply:
@duke7553 https://github.com/files-community/Files/pull/3861#discussion_r589751921 morover, we already have `FolderSettingsViewModel` in EventArgs...